### ISSUE TITLE
Error with SQLite3 Repository when inserting data

### ISSUE THREAD

**Issue by jomendoz** [fridgerator/crecto#63]
Hello,

I got the message 

> Exception: cast from String to (Time | Nil) failed

 when I try to insert a model object into SQLite3 db.

I set up everything as your README said but I always get that error. Any insight?

My model is
```crystal
module Service
  module Entities
    class Transaction < Crecto::Model
      schema "transactions" do
        field :uuid, String
        field :user, String
      end
    end
  end
end
```

My repo is
```crystal
module Service
  module Repo
    extend Crecto::Repo

    config do |conf|
      conf.adapter = Crecto::Adapters::SQLite3
      conf.database = "./development.db"
    end
  end
end
```

My schema is
```sql
DROP TABLE IF EXISTS transactions;

CREATE TABLE transactions(
    id INTEGER NOT NULL PRIMARY KEY,
    uuid varchar(255) NOT NULL,
    user varchar(255) NOT NULL,
    created_at DATETIME,
    updated_at DATETIME
);
```

Finally, I try to save a tuple with this snippet
```crystal
t = Service::Entities::Transaction.new
t.uuid = data["uuid"].to_s
t.user = data["user"].to_s
changeset = Service::Repo.insert(t)
changeset.changes.to_json
```

Full stacktrace is [here](https://pastebin.com/Q73KitcE)

**Comment by jomendoz** [comment]
I sorted out the issue changing origin repo from crystal-lang/cystal-sqlite3 v0.8.2 to Zhomart/crystal-sqlite3. I couldn't have found this problem without reading shard.yml from crecto repo.

**Comment by fridgerator** [comment]
oooooooh right, waiting on that PR to go into crystal-sqlite

Sorry about that!

**Comment by fridgerator** [comment]
just bumped the crystal-sqlite3 PR

**Comment by fridgerator** [comment]
fixed with [crystal-db v 0.4.1](https://github.com/crystal-lang/crystal-db/releases/tag/v0.4.1)
